Views/contact1.php

```diff
@@ -1,178 +1,169 @@
 <?php
 include 'header.php';
-?>
-
+   ?>
+  
 <body>
 
-  <div id="container">
+<div id="container">
     <!--Section: Contact v.2-->
-    <section class="mb-4">
+<section class="mb-4">
 
-      <!--Section heading-->
-      <h1 class="h1-responsive font-weight-bold my-4">Contact Me</h1>
-      <!--Section description-->
-      <p class="w-responsive mx-auto mb-5">Do you have any questions? Please do not hesitate to contact me. I will respond to you within
+    <!--Section heading-->
+    <h1 class="h1-responsive font-weight-bold my-4">Contact Me</h1>
+    <!--Section description-->
+    <p class="w-responsive mx-auto mb-5">Do you have any questions? Please do not hesitate to contact me. I will respond to you within
         72 hours.</p>
 
-      <div class="row">
+    <div class="row">
 
         <!--Grid column-->
         <div class="col-md-9 mb-md-0 mb-5">
-          <form id="contact-form" name="contact-form" action="" method="POST">
```

Comment:
please do not delete this line:
<form id="contact-form" name="contact-form" action="" method="POST">